apps/app-article-server/src/browser/index.css

```diff
@@ -1,21 +1,27 @@
 @font-face {
   font-family: "Roboto";
-  src: url("https://cdn.ksfmedia.fi/assets/fonts/Roboto-Regular.ttf");
+  src: url("https://cdn.ksfmedia.fi/assets/fonts/Roboto-Regular-webfont.woff2") format("woff2");
```

Comment:
We could also include both of the fonts here and let the browser decide.

The format is something around:
``` 
src:  url('Montserrat-Regular.woff') format('woff'), 
         url('Montserrat-Regular.ttf')  format('truetype'),
         url('Montserrat-Regular.svg#Montserrat') format('svg');
```

Reply:
True, I'll add that.